curs/chap-07-functii/01-empty-func/Makefile

```
@@ -0,0 +1,14 @@
+CFLAGS = -Wall -m32 -fno-stack-protector -fomit-frame-pointer -fno-PIC
```

```suggestion
CFLAGS = -Wall -m32 -fno-stack-protector -fno-PIC
```

From the commit message, I saw that this should be a demo to show stack frames. I don't know if it's intended to omit that frame pointer and show only with esp, but if it's the first demo I think it's better to have also ebp included.

---

Yes, it's intended. For the first demos I do want to have the frame pointer omitted. I want to keep it as simple as possible and introduce the frame pointer as a way to ease the addressing of local variable and function arguments.